src/app/style/styles.js

```diff
@@ -0,0 +1,67 @@
+export default {
+  header: {
+    color: "#1297A1",
+    fontFamily: "Roboto Condensed",
+  },
+  body: {
+    padding: "0 100px",
+    flex: "3",
```

Comment:
You can use numeric literals instead of string literals in a bunch of
places: `{flex: 3}`, `{height: 60}`, `{paddingLeft: 0}`, etc. My
understanding is that this is the preferred style.